charmcraft.yaml

```
@@ -49,22 +49,53 @@ requires:
 
 config:
   options:
+    cni-type:
+      type: string
+      default: bridge
+      description: |
+        Multus CNI plugin to use for the interfaces.
+        Allowed values are `bridge`, `macvlan`.
     f1-interface-name:
       type: string
-      default: "eth0"
-      description: Name of the network interface handling communication over the F1 interface.
+      default: "f1"
+      description: |
+        Name of the network interface handling communication over the F1 interface.
+        Host interface to use for the DU Network. If not specified, a bridge will be used.
+    f1-ip-address:
+      type: string
+      default: "192.168.251.7/24"
+      description: CU F1 interface IP Address
     f1-port:
       type: int
       default: 2153
       description: Number of the port handling communication over the F1 interface.
     n2-interface-name:
       type: string
-      default: "eth0"
-      description: Name of the network interface handling communication over the N2 interface.
+      default: "n2"
+      description: |
+        Name of the network interface handling communication over the N2 interface.
+        Host interface to use for the Core Network. If not specified, a bridge will be used.
```

```suggestion
        If not specified, a bridge will be used.
```
Do we need this part at all? In the first line we explain that it's the name of the interface used for N2 communication.

---

Done